javascript/javascript3/week2/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0px;
+  background-image: linear-gradient(to right, #decba4, #3e5151);
+}
+h1 {
+  width: 100%;
+  text-align: center;
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 1px;
+  padding: 0 0 40px;
+  border-bottom: double #555;
+  text-shadow: 1px 1px 2px #3e5151;
+}
+.formOne,
```

Comment:
Could this be one class `form` ?

Reply:
It could, but it was easier for me to manage between divs